src/index.html

```diff
@@ -5,9 +5,282 @@
+        <button class="burger__menu burger__menu-line">
+          <span></span>
+        </button>
+        <div class="logo header__logo"></div>
+        <a href = "tel:+380959477076" class="header__phone">+1 234 555-55-55</a>
+        <a href = "tel:+380959477076" class="header__action">Call to order</a>
+      </div>
+      <div class="description header__description">
+        <span class="description__title">Miami</span>
+        <span class="description__suptitle">Condo Kings</span>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item-active">
+            <a href="#" class="nav__link">
+              Home
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#aboutus" class="nav__link">
+              About us
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">
+              Gallery
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">
+              Consultation
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonals" class="nav__link">
+              Testimonials
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">
+              Contcats
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="miami">
+        <a name="aboutus"></a>
+        <div class="info">
+          <span class="info__heading">Miami <strong>Best</strong> Real Estate <span class="info__heading info__heading-color">here</span></span>
+          <article class="article">
+            <div class="article__title">
+              <h3 class="about">Over 50 years expirience in love</h3>
+            </div>
+            <p class="paragraph">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p class="paragraph">
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p class="paragraph"><strong>We will never be late!</strong></p>
+            <p class="paragraph"><strong>secure transaction - 100%</strong></p>
+          </article>
+        </div>
+      </section>
+      <section class="section">
+        <a name="gallery"></a>
+        <h2 class="page__title">Gallery</h2>
+        <div class="galerry">
+          <div class="photos photos__wide-rigth">
```

Comment:
это блок с модификатором
```suggestion
          <div class="photos photos--wide-rigth">
```